Libraries/LibGUI/GFilePicker.h

```diff
@@ -7,22 +7,41 @@ class GLabel;
 
 class GFilePicker final : public GDialog {
 public:
-    GFilePicker(const StringView& path = "/", CObject* parent = nullptr);
+    enum class Mode {
+        Open,
+        Save
+    };
+
+    static String get_open_filepath();
+    static String get_save_filepath();
+
+    GFilePicker(Mode type = Mode::Open, const StringView& path = "/", CObject* parent = nullptr);
     virtual ~GFilePicker() override;
 
     FileSystemPath selected_file() const { return m_selected_file; }
 
-    virtual const char* class_name() const override { return "GFilePicker"; }
-
 private:
     void set_preview(const FileSystemPath&);
     void clear_preview();
 
-    GTableView* m_view { nullptr };
+    const String ok_button_name()
```

Comment:
`const` is not really meaningful on the return type here.
Also, since the function does not need access to any `GFilePicker` instance variables, it should be `static`.